src/public/services/AuthService.ts

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios'
```

Comment:
nit: pls call this `PublicFormAuthService`, `AuthService` will be extremely confusing